### PR TITLE
Add misspellings of "transition" variations

### PR DESCRIPTION
**Pull request by FloEdelmann**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -31604,8 +31604,10 @@ transprts->transports
 transpsition->transposition
 transsend->transcend
 transtion->transition
+transtioned->transitioned
 transtions->transitions
 transtition->transition
+transtition->transitioned
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:31610]: This word `transtition` is already in the list right above it. What you are proposing is  `transtition->transition, transitioned,` ?

Comment (FloEdelmann) [reply codespell_lib/data/dictionary.txt]: Ah, I meant `transtitioned` → `transitioned`. I've fixed it in [`9c83565`](https://github.com/codespell-project/codespell/pull/2288/commits/9c835657f104f08b6b71ccdcf826292fff769ad0).

**Review by luzpaz — APPROVED** [review]

**Review by peternewman — APPROVED** [review]
LGTM, there are some other transitioned and transitioning variations missing if you want to add them @FloEdelmann ?

**Comment by FloEdelmann** [comment]
Which other variations are you thinking about?

**Comment by luzpaz** [comment]
@peternewman please re-review when you have a moment. thanks!

**Review by peternewman — APPROVED** [review]

**Comment by peternewman** [comment]
@FloEdelmann do you fancy adding the -ing variants of:
```
transision->transition
transisions->transitions
transisition->transition
transistion->transition
transistions->transitions
transitionned->transitioned
transiton->transition
transitons->transitions
transision->transition
transisions->transitions
transisition->transition
transistion->transition
transistions->transitions
transitionned->transitioned
transiton->transition
transitons->transitions
```

**Review by peternewman — APPROVED** [review]
LGTM, just if you want to add more versions.

**Review by peternewman — APPROVED** [review]

**Pull request merged**